pycbc/waveform/bank.py

```diff
@@ -490,10 +490,10 @@ def __init__(self, filename, sample_rate, minimum_buffer,
         super(LiveFilterBank, self).__init__(filename, approximant=approximant,
                 parameters=parameters, **kwds)
         self.ensure_standard_filter_columns(low_frequency_cutoff=low_frequency_cutoff)
-        self.hash_lookup = {}
+        self.param_lookup = {}
         for i, p in enumerate(self.table):
-            hash_value =  hash((p.mass1, p.mass2, p.spin1z, p.spin2z))
-            self.hash_lookup[hash_value] = i
+            key =  (p.mass1, p.mass2, p.spin1z, p.spin2z)
+            self.param_lookup[key] = i
```

Comment:
Just for good practice against crazy banks, could you add this:
```
assert(key not in self.param_lookup)
```

Reply:
good idea